kubeadm: include a couple of workarounds in kubeadm-certs.md (#16726)
kubernetes/website: content/en/docs/tasks/administer-cluster/kubeadm/kubeadm-certs.md

`kubelet.conf` is not included in the list above because kubeadm configures kubelet for automatic certificate renewal.
{{< /note >}}

{{< warning >}}
On nodes created with `kubeadm init`, prior to kubeadm version 1.17, there is a

> [bug](https://github.com/kubernetes/kubeadm/issues/1753) where you manually have to modify the contents of `kubelet.conf`. After `kubeadm init` finishes, you should update `kubelet.conf` to point to the

Reviewer: I think we should specify that this problem happens only on the node where kubeadm init is executed

Author: @fabriziopandini 
i think the warning already clarifies that:
> On nodes created with `kubeadm init`
?

`kubelet.conf` is not included in the list above because kubeadm configures kubelet for automatic certificate renewal.
{{< /note >}}

{{< warning >}}
Prior to kubeadm version 1.17 there is a [bug](https://github.com/kubernetes/kubeadm/issues/1753)
where you manually have to modify the contents of `kubelet.conf` on nodes where `kubeadm init`

> was called to point to the rotated kubelet client certificates after `kubeadm init` finishes.

Reviewer 2: ```suggestion
was called. You should update `kubelet.conf` to point to the rotated kubelet client certificates after `kubeadm init` finishes, then
```
something like this?

Author: will split the sentences.

Author: updated.

If you have more complex requirements for certificate renewal, you can opt out from the default behavior by passing `--certificate-renewal=false` to `kubeadm upgrade apply` or to `kubeadm upgrade node`.

{{< warning >}}
Prior to kubeadm version 1.17 there is a [bug](https://github.com/kubernetes/kubeadm/issues/1818)
where the default value for `--certificate-renewal` is `false` for the `kubeadm upgrade node`

> command, so the flag has to be passed explicitly `--certificate-renewal=true`.

Reviewer 2: ```suggestion
command. In that case, you should explicitly set `--certificate-renewal=true`.
```
(aim here is to write in the active voice).
